### DISCUSSION TITLE
3rd Parties that offer support for M1

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2977] [Extensions & Modules]
A few companies that I have identified as still offering support for M1

**Webscale**
https://docs.webscale.com/reference/m1eol-patches/

**Mage One**
https://mage-one.com/

**Magentary**
https://magentary.com/magento-patch-installation-service/
https://magentary.com/faq-category/magento-maintenance/

**Comment by sreichel** [comment]
Would be nice to have access to that patches ... :/

**Comment by Flyingmana** [comment]
* https://www.nexcess.net/help/safe-harbor-for-magento-1/
* https://maxcluster.de/blog/2020/7/sicheres-hosting-magento1-shops
